modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/view_flat_users.jsp

```diff
@@ -29,6 +29,7 @@ String displayStyle = viewFlatUsersDisplayContext.getDisplayStyle();
 %>
 
 <clay:management-toolbar
+	itemType='<%= LanguageUtil.get(request, "users") %>'
```

Comment:
I may be wrong, but `users-admin-web` is not owned by Lima? If so this should be sent to them or at least mentioned here for it to be reviewed since it's affecting to their module.

Reply:
This module belongs to Users and System Management. Right now, from the PEDS team we are managing this module.